index.html

```diff
@@ -0,0 +1,263 @@
+          faster. He is the...
+        </p>
+      </div>
+      <div class="items1">
+        <h2>Person4</h2>
+        <p>
+          Andy Davies is an independent web performance consultant who helps
+          retailers, publishers, and financial services make their websites
+          faster. He is the...
+        </p>
+      </div>
+      <div class="items2">
+        <h2>Person5</h2>
+        <p>
+          Andy Davies is an independent web performance consultant who helps
+          retailers, publishers, and financial services make their websites
+          faster. He is the...
+        </p>
+      </div>
+      <div class="items3">
+        <h2>Person6</h2>
+        <p>
+          Andy Davies is an independent web performance consultant who helps
+          retailers, publishers, and financial services make their websites
+          faster. He is the...
+        </p>
+      </div>
+    </div>
+    <div class="systems flex lower-bar items3">
+      <h1>FEATURED BOOK</h1>
+      <p>
+        Meet Design Systems, a recent Smashing book in which Alla Kholmatova
+        explores how to set up an effective design system to create great
+        digital products. With common traps, gotchas and lessons learned.
+      </p>
+    </div>
+    <div class="article-post flex top-bar">
+      <div class="items2">
+        <h1>New Front-End Challenges To Master</h1>
+        <ul>
+          <li><a href="">item1</a></li>
+          <li><a href="">item2</a></li>
+          <li><a href="">item3</a></li>
+          <li><a href="">item4</a></li>
+        </ul>
+      </div>
+      <div class="items1">
+        <h1>Performance Optimization & Strategy</h1>
+        <ul>
+          <li><a href="">item1</a></li>
+          <li><a href="">item2</a></li>
+          <li><a href="">item3</a></li>
+          <li><a href="">item4</a></li>
+        </ul>
+      </div>
+      <div class="items2">
+        <h1>Improving Workflow & Data Protection</h1>
+        <ul>
+          <li><a href="">item1</a></li>
+          <li><a href="">item2</a></li>
+          <li><a href="">item3</a></li>
+          <li><a href="">item4</a></li>
+        </ul>
+      </div>
+      <div class="items3">
+        <h1>Design and partner Checklist</h1>
+        <ul>
+          <li><a href="">item1</a></li>
+          <li><a href="">item2</a></li>
+          <li><a href="">item3</a></li>
+          <li><a href="">item4</a></li>
+        </ul>
+      </div>
+      <div class="large-boxes items4">
+        <h1>Smashing Newsletter</h1>
+      </div>
+    </div>
+    <footer class="items4">
+      <p>Browse All Topics</p>
+      <ul>
+      <li><a href="#">link1</a></li>
+      <li><a href="#">link2</a></li>
+      <li><a href="#">link3</a></li>
+      <li><a href="#">link4</a></li>
+      <li><a href="#">link5</a></li>
+      <li><a href="#">link6</a></li>
+      <li><a href="#">link7</a></li>
+      </ul>
+    </footer>
+  </body>
```

Comment:
Add more links in the footer and center them in the same fashion as in the original website